src/styles/_catalog.scss

```diff
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
More correctly move repeated value to variable, so that if you need to change the value of the width of the card, then it will be enough to change one variable, and not search through the entire code